frontend/packages/core/src/NPS/feedback.tsx

```diff
@@ -185,21 +204,42 @@ const NPSFeedback = (opts: FeedbackOptions) => {
         container
         direction="row"
         alignItems="center"
-        justify="center"
-        style={{ padding: "16px" }}
+        style={{ padding: wizardOrigin ? "16px" : "32px" }}
       >
-        <MuiGrid item xs={6}>
-          <Typography variant="subtitle3">{survey.prompt}</Typography>
+        <MuiGrid item xs>
+          <Typography variant={wizardOrigin ? "subtitle3" : "subtitle2"}>
+            {survey.prompt}
+          </Typography>
         </MuiGrid>
-        <MuiGrid item xs={6} style={{ display: "flex", justifyContent: "space-around" }}>
-          <EmojiRatings ratings={survey.ratingLabels} setRating={setSelected} />
+        <MuiGrid
+          item
+          xs={wizardOrigin ? 6 : 12}
+          style={{ margin: "8px 0px", display: "flex", justifyContent: "space-around" }}
+        >
+          <EmojiRatings
+            ratings={survey.ratingLabels}
+            setRating={setSelectedEmoji}
+            placement={wizardOrigin ? "top" : "bottom"}
+            size={wizardOrigin ? "small" : "large"}
+          />
         </MuiGrid>
-        {selected !== null && (
+        {selectedEmoji !== null && (
           <>
+            {!wizardOrigin && options.feedbackTypes && (
+              <MuiGrid item xs={12} style={{ margin: "32px 0px 16px 0px" }}>
+                <Select
+                  name="anytimeSelect"
+                  label="Choose a type of feedback you want to submit"
+                  options={options.feedbackTypes}
+                  onChange={setFeedbackType}
+                />
+              </MuiGrid>
+            )}
             <MuiGrid item xs={12}>
               <StyledTextField
                 multiline
                 fullWidth
+                $origin={origin}
```

Comment:
out of curiosity, what does the `$` do?

Reply:
This is part of the change added via #1944. It makes sure the transient property doesn't get out to the dom.